app/javascript/react/components/Navbar/Navbar.style.tsx

```diff
@@ -10,16 +10,24 @@ import {
 } from "../../assets/styles/colors";
 import { media } from "../../utils/media";
 
+const NAVBAR_HEIGHT = "7.8rem";
+
 const Header = styled.header`
   display: flex;
   justify-content: space-between;
   align-items: center;
-  padding: 2rem;
-  background: ${gray100};
+  background-color: transparent;
+  z-index: 2;
+  flex-wrap: wrap;
+  position: absolute;
```

Comment:
Are we confident that this change won't impact the current AirCasting page? I'm just double-checking since I'm not fully familiar with the feature flag setup. Is the current page built entirely in Elm, meaning there are no concerns to worry about?

Reply:
I think it is ok. React App is not connected yet with the production version